internal/provisioner/teleport.go

```diff
@@ -103,15 +103,6 @@ func (n *teleport) Destroy() error {
 		return errors.Wrap(err, "unable to delete Teleport bucket")
 	}
 
-	err = n.awsClient.DynamoDBEnsureTableDeleted(teleportClusterName, n.logger)
```

Comment:
@Szymongib Can you remove the lines above as well that remove the Teleport bucket?

Reply:
@stylianosrigas Sure, I was considering this however there is no permission issue and S3 buckets still exist for old clusters so I decided to leave it as it is so that the bucket will be cleaned up when a cluster is deleted.